### PR TITLE
minor configuration improvements

### PR DESCRIPTION
**Pull request by onursumer**
- disabled the admin controller for now
- excluded conflicting log4j dependencies from biojava

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,7 +22,7 @@
     </parent>
 
     <properties>
-      <java.version>1.8</java.version>
```
Comment (sheridancbio) [inline pom.xml:25]: Not sure why we are reducing to java 1.7 ... do we need to test all the modules and make sure nothing breaks?

**Review by sheridancbio — COMMENTED** [review]
I think we should test all modules before reducing jvm to 1.7.

File: pom.xml (hunk 1 above)
Comment (onursumer) [reply pom.xml:25]: I was trying to make the build compatible with the current tomcat 7 & java 7 configuration on MSK servers. I am not sure we use anything java 1.8 specific in the current code base. We may change it back to 1.8 if needed.

Comment (sheridancbio) [reply pom.xml:25]: ok ... merging as is. If something breaks we will switch back.

**Pull request merged**